pages/reference/cli.md

```diff
@@ -1377,28 +1383,32 @@ Docker host TLS key file
 
 ## push &#60;applicationOrDevice&#62;
 
-This command can be used to start a build on the remote balena cloud builders,
-or a local mode balena device.
+This command can be used to start an image build on the remote balenaCloud build
+servers, or on a local-mode balena device.
 
-When building on the balena cloud the given source directory will be sent to the
-balena builder, and the build will proceed. This can be used as a drop-in
-replacement for git push to deploy.
+When building on the balenaCloud servers, the given source directory will be
+sent to the remote server. This can be used as a drop-in replacement for the
+"git push" deployment method.
 
-When building on a local mode device, the given source directory will be built
-on the device, and the resulting containers will be run on the device. Logs will
-be streamed back from the device as part of the same invocation.
+When building on a local-mode device, the given source directory will be
+built on the device, and the resulting containers will be run on the device.
+Logs will be streamed back from the device as part of the same invocation.
+The web dashboard can be used to switch a device to local mode:
+https://www.balena.io/docs/learn/develop/local-mode/
 
 The --registry-secrets option specifies a JSON or YAML file containing private
 Docker registry usernames and passwords to be used when pulling base images.
 Sample registry-secrets YAML file:
 
-	'https://idx.docker.io/v1/':
-		username: mike
-		password: cze14
-	'myregistry.com:25000':
+	'my-registry-server.com:25000':
 		username: ann
 		password: hunter2
-
+	'':  # Use the empty string to refer to the Docker Hub
+		username: mike
+		password: cze14
+	'eu.gcr.io':  # Google Container Registry
+		username: '_json_key'
+		password: '{escaped contents of the GCR keyfile.json file}'
```

Comment:
I don't know why these changes are appearing in this pull request, because they are already reflected in the following page: https://www.balena.io/docs/reference/cli/
Again, all I did was to run `npm i` that in turn runs `./tools/prepare.sh` that in turn runs `./tools/fetch-external.sh` that fetched the updated reference docs for the CLI and Python SDK.